execnet/gateway_io.py

```diff
@@ -67,8 +67,19 @@ def killpid(pid):
 popen_bootstrapline = "import sys;exec(eval(sys.stdin.readline()))"
 
 
+def shell_split_path(path):
+    """
+    Use shell lexer to split the given path into a list of components,
+    taking care to handle Windows' '\' correctly.
+    """
+    if sys.platform.startswith('win'):
+        # replace \\ by / otherwise shlex will strip them out
+        path = path.replace('\\', '/')
+    return shlex.split(path)
+
+
 def popen_args(spec):
-    args = shlex.split(spec.python) if spec.python else [sys.executable]
```

Comment:
It seems this was a bug on Windows:

```
>>> import shlex
>>> shlex.split(r'X:\execnet\.env36\Scripts\python.exe')
['X:execnet.env36Scriptspython.exe']
```
Which of course is not a valid Python file. My quick and dirty solution at least gives a valid file:

```
import shlex
>>> shlex.split(r'X:\execnet\.env36\Scripts\python.exe'.replace('\\', '/'))
['X:/execnet/.env36/Scripts/python.exe']
```